tests/python/tools/test_profiler.py

```diff
@@ -60,6 +61,7 @@ def test_profiler_with_cuda(i):
         loss.backward()
         profiler.stop()
     data = profiler.get()
+    profiler.clear()
     assert len(data["traceEvents"]) >= 0
```

Comment:
Add one more assert before clear to make sure the events are still cached.

Reply:
Thanks, will add after fixing tests.